exercises/practice/pop-count/test-pop-count.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+# generated on 2023-10-18T13:03:37Z
+load bats-extra
+
+@test '0 eggs' {
+    #[[ $BATS_RUN_SKIPPED == "true" ]] || skip
+
+    run jq -r -f pop-count.jq << 'END_INPUT'
+{
+  "number": 0
+}
+END_INPUT
```

Comment:
```suggestion
    run jq -r -f pop-count.jq << '{"number": 0}'
```

Reply:
I don't want to change this: this results from the `bin/generate_tests` script that simply grabs the input from the problem-specs canonical-data.json.